_docs/7_SubsquidArchive/archive.subspace.network

```diff
@@ -1,14 +1,18 @@
 server {
-    root /var/www/arhive.subspace.network/html;
+    root /var/www/html;
     index index.html index.htm index.nginx-debian.html;
 
     server_name archive.subspace.network;
 
     location / {
-        try_files $uri $uri/ =404;
+        try_files $uri $uri/ @backend;
     }
```

Comment:
Why do we even have default location here?

Reply:
GraphQL explorer UI requires some static files from `http://127.0.0.1:4444` - css, js...

Reply:
I see. It would be ideal to specify those subdirectories with static files explicitly though, just to be on a safe side and not serve `.env` file accidentally or whatnot. If there isn't anything like that in that location then it is not blocking.